source/JumpTypes.h

```diff
@@ -0,0 +1,27 @@
+/* JumpTypes.h
+Copyright (c) 2022 by Amazinite
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef JUMP_TYPES_H_
+#define JUMP_TYPES_H_
+
+enum class JumpType : int {
+	HYPERDRIVE,
```

Comment:
According to quyykk, no caps:
https://github.com/endless-sky/endless-sky/pull/7213#discussion_r990455565

Reply:
I don't know why @quyykk said no caps, I was under the impression that we did caps these things.

Reply:
If you say so, I do not know for sure.

Reply:
There are several `enum class`es where the values aren't all caps, but most are.